content/es/security/code_security/static_analysis/static_analysis_rules/_index.md

```diff
@@ -2,257 +2,356 @@
-      Reglas centradas en encontrar problemas de seguridad en tu código de JavaScript.
+    description: 'Reglas centradas en encontrar problemas de seguridad en tu código
+      de JavaScript.
+
+      '
   javascript-express:
     title: Consultar las prácticas recomendadas y seguridad de Express.js
-    description: |
-      Reglas específicas para las prácticas recomendadas y seguridad de Express.js.
+    description: 'Reglas específicas para las prácticas recomendadas y seguridad de
+      Express.js.
+
+      '
   javascript-inclusive:
     title: Verifica el código de JavaScript para detectar problemas de redacción.
-    description: |
-      Reglas de JavaScript para evitar redactar textos inadecuados en el código y los comentarios.
+    description: 'Reglas de JavaScript para evitar redactar textos inadecuados en
+      el código y los comentarios.
+
+      '
   javascript-node-security:
     title: Identificar posibles puntos críticos de seguridad en Node
-    description: |
-      Reglas para identificar posibles puntos críticos de seguridad en Node. Esto puede incluir falsos positivos que requieren una evaluación más exhaustiva.
+    description: 'Reglas para identificar posibles puntos críticos de seguridad en
+      Node. Esto puede incluir falsos positivos que requieren una evaluación más exhaustiva.
+
+      '
   jsx-react:
     title: Reglas de linting específicas de React
-    description: |
-      Este complemento exporta una configuración `recommended` que aplica las prácticas recomendadas de React.
+    description: 'Este complemento exporta una configuración `recommended` que aplica
+      las prácticas recomendadas de React.
+
+      '
   kotlin-best-practices:
     title: Seguir las mejores prácticas para escribir código de Kotlin
-    description: |
-      Reglas para aplicar las prácticas recomendadas de Kotlin.
+    description: 'Reglas para aplicar las prácticas recomendadas de Kotlin.
+
+      '
   kotlin-code-style:
     title: Aplicar el estilo de código de Kotlin
-    description: |
-      Reglas para aplicar el código de estilo de Kotlin.
+    description: 'Reglas para aplicar el código de estilo de Kotlin.
+
+      '
   kotlin-security:
     title: Aplicar la codificación segura de Kotlin
-    description: |
-      Reglas enfocadas a encontrar problemas de seguridad en tu código Kotlin.
+    description: 'Reglas enfocadas a encontrar problemas de seguridad en tu código
+      Kotlin.
+
+      '
   php-best-practices:
     title: Seguir las mejores prácticas para escribir código de PHP
-    description: |
-      Reglas para aplicar las prácticas recomendadas de PHP, mejorar el estilo de código, prevenir errores, y promover un código de PHP de alto rendimiento, fácil de mantener y eficiente.
+    description: 'Reglas para aplicar las prácticas recomendadas de PHP, mejorar el
+      estilo de código, prevenir errores, y promover un código de PHP de alto rendimiento,
+      fácil de mantener y eficiente.
+
+      '
```

Comment:
@davidejones Just wanted to confirm that the blank lines in these descriptions are OK?

Reply:
Thanks, yea it shouldn't have changed those. I've removed those changes. 🙇